### ISSUE TITLE
Explicit Label and describedby causing violation

### ISSUE THREAD

**Issue by TristanCooper** [NickColley/jest-axe#222]
I have a checkbox with `aria-describedby` like so:

```          
<CSS.CheckboxWrapper>
  <Checkbox
    colour={colour}
    onChange={() => onClick(staticId)}
    aria-checked={Boolean(selected)}
    aria-describedby={ariaDescribedBy}
    checked={Boolean(selected)}
    id={`${staticId}_checkbox`}
    checkboxStyle={checkboxStyle}
    data-testid="changePlanOption-checkbox"
    />
</CSS.CheckboxWrapper>
```

and its label with the for/htmlFor attribute like so:

```
<Text data-testid="changePlanOption-label" variant="titleMedium" as="label" htmlFor={`${staticId}_checkbox`}>
```

My understanding is that I can have both the label like this and the aria-describedby on my Checkbox component.

However I receive the following error when running the unit tests:
`"Form elements should have a visible label (label-title-only)"`
Where removing the aria-describedby attribute makes the test pass. If my understanding is correct then having both the describedby and the explicit label should not cause any violation.

**Comment by NickColley** [comment]
Just published 7.0.0 please try your issue in 7.0.0 and re-open if it's an issue, thanks a lot.
